packages/beacon-node/src/chain/validation/blobSidecar.ts

```diff
@@ -18,6 +18,15 @@ export async function validateGossipBlobSidecar(
 ): Promise<void> {
   const blobSlot = blobSidecar.signedBlockHeader.message.slot;
 
+  // [REJECT] The sidecar's index is consistent with `MAX_BLOBS_PER_BLOCK` -- i.e. `blob_sidecar.index < MAX_BLOBS_PER_BLOCK`.
+  if (blobSidecar.index < chain.config.MAX_BLOBS_PER_BLOCK) {
```

Comment:
just noticed the check is incorrect, this should be 
```ts
blobSidecar.index >= chain.config.MAX_BLOBS_PER_BLOCK
```

Reply:
Oops